_dprhtml/js/index.js

```diff
@@ -39,10 +39,7 @@ async function mainInitialize() {
   } else if (DPR_PAL.isDictionaryFeature()) {
     await loadFeatureAsync(sectionId, dictionaryFeatureName, initializeDictionaryFeature);
   } else {
-    await loadHtmlFragmentAsync("#main-content-landing-page", 'features/landing-page/main-pane.html');
-    __dprViewModel.showLandingFeature();
-    initFeedbackFormParameters();
-    await DPR_bv_mod.showBv();

```

Comment:
I needed to reuse this bit of code when the browser history state is changed, so I just moved it into a new function.

Note that we cannot just call `__dprViewModel.showLandingFeature()` when the browser history state changes, because if at some point the user refreshes the page, some resources may not yet be loaded. For example, let's say the user is looking at some Pāli text, refreshes the page, and then tries to navigate back to the home page. The `features/landing-page/main-pane.html` fragment is not yet loaded, so only calling `__dprViewModel.showLandingFeature()` will result in a blank screen.